source/PlayerInfo.cpp

```diff
@@ -303,11 +304,18 @@ void PlayerInfo::Load(const string &path)
 				if(grand.Size() >= 2)
 					costBasis[grand.Token(0)] += grand.Value(1);
 		}
+		// This node is called "stock" and not "outfit stock" for backwards compatibility.
```

Comment:
Alternatively, we could load from both "stock" and "outfit stock", but save only as "outfit stock". This way it would be clear that it's for outfits on new and updated saves, while still being able to read old saves.